### PR TITLE
Update hic mapping

### PR DESCRIPTION
**Pull request by ksenia-krasheninnikova**
HiC mapping from 

https://github.com/sanger-tol/treeval/blob/80554a803903183613d49690d5770eeadb3c42c9/subworkflows/local/hic_mapping.nf

was transferred to the main pipeline.

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/genomeassembly/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the sanger-tol/genomeassembly _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/hic_mapping.nf [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                                  tuple([ 
+                                        id: cram_id.id
+                                        ], 
+                                    file(cram_info[0]),
+                                    cram_info[1],
+                                    cram_info[2],
+                                    cram_info[3],
+                                    cram_info[4],
+                                    cram_info[5],
+                                    cram_info[6],
```
Comment (priyanka-surana) [inline subworkflows/local/hic_mapping.nf:66]: Can you add comments to describe what each of these fields actually contains?

**Review by priyanka-surana — COMMENTED** [review]
I want to test this after we have a clean template merge. Then happy to approve.

File: subworkflows/local/hic_mapping.nf (hunk 1 above)
Comment (ksenia-krasheninnikova) [reply subworkflows/local/hic_mapping.nf]: Done.

**Review by priyanka-surana — APPROVED** [review]
LGTM 👍🏼 The following test worked for me:
```
nextflow run . -ansi-log false -profile test,singularity,longranger_lsf_sanger --outdir results
```

**Pull request merged**